libs/wire-api/src/Wire/API/Team/Feature.hs

```diff
@@ -267,6 +271,52 @@ instance HasDeprecatedFeatureName 'TeamFeatureDigitalSignatures where
 typeTeamFeatureName :: Doc.DataType
 typeTeamFeatureName = Doc.string . Doc.enum $ cs . toByteString' <$> [(minBound :: TeamFeatureName) ..]
 
+----------------------------------------------------------------------
+-- TeamFeatureTTLValue
+
+-- Using Word to avoid dealing with negative numbers.
+-- Ideally we would also not support zero.
+-- Currently a TTL=0 is ignored on the cassandra side.
+data TeamFeatureTTLValue
+  = TeamFeatureTTLDays Word
```

Comment:
Do we have any natural implementation in use? Is it worth going for more type-level constriction here or documenting externally that `TTL=0` is ignored?